Software/backend/database/MongoDB/Main.md

```diff
@@ -0,0 +1,42 @@
+<!-- press ctrl+K then V to open a preview of the MarkDown file  -->
+# MonogoDB
+MongoDB is a cross-platform document-oriented database program. 
+Classified as a NoSQL database program, MongoDB uses JSON-like documents with optional schemas. 
+MongoDB is developed by MongoDB Inc. and licensed under the Server Side Public License (SSPL).
+
```

Comment:
as this is a tutorial, best move this description into the intro.
here maybe say that what this tutorial is for generally- "learn to use mongodb" for instance.